tools/metrics/apk_size.py

```diff
@@ -7,8 +7,9 @@
 from os import path, listdir, stat
 from sys import exit
 
-SIZE_LIMIT = 5242880
+SIZE_LIMIT = 5
```

Comment:
nit: we should specify units here to make sure no one uses the wrong ones, e.g. `SIZE_LIMIT_MB`.

Then `BYTE_LIMIT` should match a similar pattern. Maybe, `SIZE_LIMIT_BYTES`.